bundled_scripts/ilo.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
```

Comment:
I've been thinking about testing these scripts too, but as you've noticed, running Python's tests with Golang's may be tricky/complicated (if not impossible) on e.g. Travis.

And speaking of e2e tests - yes, that may be a sensible way to approach above problem - I'll create a ticket for that.